helm/csi-isilon/templates/csidriver.yaml

```diff
@@ -4,4 +4,4 @@ metadata:
     name: isilon
 spec:
     attachRequired: true
-    podInfoOnMountVersion: "v1"
+    podInfoOnMount: false
```

Comment:
Should podInfoOnMount be true?

Reply:
Just looked, likely it should be. When I deployed it via helmv2 on 1.16 the value was false. I made an assumption from that.